Runtimes/Core/cmake/modules/EmitSwiftInterface.cmake

```diff
@@ -26,5 +36,18 @@ function(install_swift_interface target)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.private.swiftinterface"
       RENAME "${SwiftCore_MODULE_TRIPLE}.private.swiftinterface"
       DESTINATION "${SwiftCore_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
+
+    # Install catalyst interface files
+    if(SwiftCore_VARIANT_MODULE_TRIPLE)
+      install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${target}-${SwiftCore_VARIANT_MODULE_TRIPLE}/${target}.swiftmodule"
```

Comment:
I don't know how I feel about the `install_swift_interface` installing the swift module here. We should hoist this to the install rules for the modules, or we can have a helper to install the swiftmodules.